src/main/kotlin/study/racingcar/ui/ResultView.kt

```diff
@@ -1,23 +1,23 @@
 package study.racingcar.ui
 
-import study.racingcar.domain.Car
+import study.racingcar.domain.Race
+import study.racingcar.dto.RacingCarState
 
 object ResultView {
-    fun displayCars(cars: List<Car>) {
+    fun displayCars(cars: List<RacingCarState>) {
         println("실행 결과")
         cars.forEach { car ->
             println("${car.name} : ${displayPosition(car)}")
         }
         println()
     }
 
-    fun displayWinners(cars: List<Car>) {
-        val maxPosition = cars.maxByOrNull { it.position }?.position ?: 0
-        val winners = cars.filter { it.position == maxPosition }.joinToString { it.name.name }
+    fun displayWinners(race: Race) {
```

Comment:
마찬가지로 Domain 객체를 전달하기 보단 정의해주신 DTO를 전달하는것이 어울려 보입니다 😄 
```suggestion
    fun displayWinners(cars: List<RacingCarState>) {
```

Reply:
그 부분을 제가 놓쳤군요! 꼼꼼한 리뷰 감사합니다 🙂